config/components/kueueviz/backend-deployment.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: apps/v1
```

Comment:
Is this moved or completely new? If new please add in a separate PR

Reply:
it is only a move

Reply:
Hm, but then I also see this on the diff: 
![image](https://github.com/user-attachments/assets/448ebfeb-2454-416c-ba29-1949fcedc2cd)

so is it now in two places? Asking cause I imagine git should pick up properly on moves, but it displays for me as added file...

Reply:
ok, fixed after some rebases and commit amends